perfkitbenchmarker/providers/aws/util.py

```diff
@@ -23,6 +26,21 @@
 FLAGS = flags.FLAGS
 
 
+def IsRegion(zone_or_region):
+  """Returns whether "zone_or_region" is a region."""
+  if not re.match('[a-z]{2}-[a-z]+-[0-9][a-z]?\Z', zone_or_region):
```

Comment:
Make this a raw string.
Nit: I think `$` is more commonly used in the codebase to identify the end of a string.